### ISSUE TITLE
duo : wdt : 编译告警

### ISSUE THREAD

**Issue by unicornx** [RT-Thread/rt-thread#8933]
`bsp/cvitek/cv18xx_risc-v` master
----

```
/home/u/ws/duo/rt-thread/bsp/cvitek/drivers/drv_wdt.c: In function '_wdt_control':
/home/u/ws/duo/rt-thread/bsp/cvitek/drivers/drv_wdt.c:119:38: warning: assignment to 'void *' from 'unsigned int' makes pointer from integer without a cast [-Wint-conversion]
  119 |         wdt_device->parent.user_data = (rt_uint32_t)(*(rt_uint32_t *)arg);
      |                                      ^
/home/u/ws/duo/rt-thread/bsp/cvitek/drivers/drv_wdt.c:122:31: warning: cast from pointer to integer of different size [-Wpointer-to-int-cast]
  122 |         *(rt_uint32_t *)arg = (rt_uint32_t)wdt_device->parent.user_data;
      |                               ^
```

**Comment by unicornx** [comment]
已经合入 duo-v5.1.0 <https://github.com/flyingcys/rt-thread/pull/33>，有待 upstream

**Comment by unicornx** [comment]
fixed in #9201
